pkgs/by-name/se/searxng/package.nix

```diff
@@ -5,13 +5,13 @@
 
 python3.pkgs.toPythonModule (python3.pkgs.buildPythonApplication rec {
   pname = "searxng";
-  version = "unstable-2023-10-31";
+  version = "unstable-2024-02-23";
```

Comment:
Should we change this to be `1.0.0-unstable-2024-02-23` to conform to the new versioning scheme?

Reply:
I don't know, they changed the versioning scheme since and are not really tagging things anymore.

Reply:
I guess then we should use `0-unstable-2024-02-23`

Reply:
```suggestion
  version = "0-unstable-2024-02-23";
```

Reply:
Doesn't work with setuptools 

```
setuptools.extern.packaging.version.InvalidVersion: Invalid version: 'unstable.2024.02.24+d72fa99b'
```